microprofile-health/src/test/java/org/jboss/eap/qe/microprofile/health/integration/FailSafeCDIHealthBaseTest.java

```diff
@@ -24,23 +24,38 @@
 import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.ArquillianContainerProperties;
 import org.jboss.eap.qe.microprofile.tooling.server.configuration.arquillian.ArquillianDescriptorWrapper;
 import org.jboss.eap.qe.microprofile.tooling.server.configuration.creaper.ManagementClientProvider;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.asset.EmptyAsset;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
 import org.junit.Before;
 import org.junit.Ignore;
 import org.junit.Test;
 import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
 
 import io.restassured.http.ContentType;
-import io.restassured.response.ValidatableResponse;
 import io.restassured.specification.RequestSpecification;
 
 /**
- * Testclass to test {@code /health}, {@code /health/live}, {@code /health/ready} endpoints. Liveness and readiness probes
- * are configured via MP Config properties at the beginning. Tests in the testclass are not expected to change state of
- * probes after the initialization in the beginning since some ways of MP Config configurations in order to change values
- * requires a WildFly to be reloaded.
+ * Testclass to test {@code /health}, {@code /health/live}, {@code /health/ready} endpoints.
+ * {@link CDIBasedLivenessHealthCheck} and {@link CDIBasedReadinessHealthCheck} probes are based on fail-sage CDI bean
+ * {@link FailSafeDummyService} configured at the beginning via MP Config properties:
+ * <ul>
+ * <li>{@link FailSafeDummyService#LIVE_CONFIG_PROPERTY} - service is live</li>
+ * <li>{@link FailSafeDummyService#READY_CONFIG_PROPERTY} - service is ready if
+ * {@link FailSafeDummyService#IN_MAINTENANCE_CONFIG_PROPERTY} is {@code false}</li>
+ * <li>{@link FailSafeDummyService#READY_IN_MAINTENANCE_CONFIG_PROPERTY} - service is ready if
+ * {@link FailSafeDummyService#IN_MAINTENANCE_CONFIG_PROPERTY} is {@code true}</li>
+ * </ul>
+ * If service is in maintenance a method simulating opening resources {@link FailSafeDummyService#simulateOpeningResources}
+ * throws and {@link IOException}.
+ * Subclasses are supposed to configure only MP Config - set MP Config properties in different ways.
+ *
+ * Tests in the testclass are not expected to change state of probes after the initialization in the beginning since some
+ * ways of MP Config configurations in order to change values requires a WildFly to be reloaded.
```

Comment:
I'd reword the final part of this sentence, for instance:
```
Tests in this test class are not expected to change the state of probes after their initialization since some
 ways of configuring MP Config would require a Wildfly reload in order to update those values.
```

But most importantly, I'd like to know whether this affects the logic of the tests that this class is executing. In this case, I'd emphasize this aspect.